org.mpo.Dayon.yml

```
@@ -0,0 +1,51 @@
+app-id: io.github.retgal.Dayon
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home:rw
```

```suggestion
  - --filesystem=home
```

still don't like it, bu `rw` is implicit.